boxsdk/util/translator.py

```diff
@@ -28,12 +28,33 @@ def _get_object_id(obj):
         `dict`
     :return:
     """
-    if obj.get('type', '') == 'event':
+    obj_type = obj.get('type')
```

Comment:
This logic probably belongs in object classes. I know this method isn't new as of this review, but it's becoming more obvious that this is the wrong place for this logic.